src/matching/matcher.ts

```diff
@@ -17,11 +17,6 @@ export class PairMatcher {
     // useful for text objects.
     "<" : { match: ">",  nextMatchIsForward: true  },
     ">" : { match: "<",  nextMatchIsForward: false },
-    // These are useful for deleting closing and opening quotes, but don't seem to negatively
-    // affect how text objects such as `ci"` work, which was my worry.
-    '"' : { match: '"',  nextMatchIsForward: false  },
```

Comment:
Actually you're right, I just duplicated that code in surround.vim. Yeahhhhhh. 

I still would like it to be here for completeness though. :)